spec/presenters/announcement_json_presenter_spec.rb

```diff
@@ -4,7 +4,7 @@
   let!(:announcement) { create :announcement }
 
   before do
-    setup_github_data
+    stub_coronavirus_landing_page_content(announcement.coronavirus_page)
```

Comment:
It looks like there's also duplications that can be cleaned up in: support/coronavirus_feature_steps.rb and controllers/reorder_sub_sections_controller_spec.rb